app/src/main/java/ch/epfl/sdp/contamination/databaseIO/GridFirestoreInteractor.java

```diff
@@ -1,19 +1,25 @@
 package ch.epfl.sdp.contamination.databaseIO;
 
 import android.location.Location;
+import android.util.Log;
 
 import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 
 import ch.epfl.sdp.contamination.Carrier;
-import ch.epfl.sdp.identity.Account;
 import ch.epfl.sdp.firestore.ConcreteFirestoreInteractor;
+import ch.epfl.sdp.identity.Account;
+
+import static ch.epfl.sdp.firestore.FirestoreLabels.LAST_POSITIONS_COLL;
+import static ch.epfl.sdp.firestore.FirestoreLabels.LIVE_GRID_COLL;
+import static ch.epfl.sdp.firestore.FirestoreLabels.TIMES_LIST_COLL;
+import static ch.epfl.sdp.firestore.FirestoreLabels.UNIXTIME_TAG;
 
 public class GridFirestoreInteractor extends ConcreteFirestoreInteractor {
 
-    // MODEL: Round the location to the 5th decimal digit
-    public static final int COORDINATE_PRECISION = 100000;
+    // MODEL: Round the location to the 3th decimal digit
```

Comment:
This constant is used to calculate the cell of the LiveGrid and is hence conceptually different from your ROUNDING_FACTOR, since many different (but close) positions map to the same LiveGrid cell

Reply:
Yep you're right ! It's fine